randovania/game_description/db/region.py

```diff
@@ -23,14 +23,10 @@ def __repr__(self):
     def dark_name(self) -> str | None:
         return self.extra.get("dark_name")
 
-    @property
-    def world_asset_id(self) -> int:
-        return self.extra["asset_id"]
-
     @property
     def all_nodes(self) -> Iterator[Node]:
         """
-        Iterates over all nodes in all areas of this world.
+        Iterates over all nodes in all areas of this db.
```

Comment:
I think this one ought to be world, not db

Reply:
pycharm renamed a bunch of comments and strings when I renamed the folder, it's weird